source/guide_invoiceninja.rst

```diff
@@ -126,6 +125,14 @@ User Details
 
 After you save the settings, you will be redirected to the login for the admin panel.
 
+We now can change back to PHP Version 7.4:
```

Comment:
Can you point me to some documentation, which says that 7.4 is supported? The [official docs](https://docs.invoiceninja.com/install.html) say only <= 7.2 is supported. Some github issues (https://github.com/invoiceninja/invoiceninja/issues/4221#issuecomment-717618019, https://github.com/invoiceninja/invoiceninja/issues/3503#issuecomment-599262500) claim support for 7.3, but explicitly rule out support for 7.4. Additionally, 7.4 seems to cause errors similar to the ones mentioned in the linked issues:
```
[16-Feb-2021 18:51:28] WARNING: [pool www] child 12282 said into stderr: "[16-Feb-2021 17:51:28 UTC] PHP Deprecated:  Unparenthesized `a ? b : c ?: d` is deprecated. Use either `(a ? b : c) ?: d` or `a ? b : (c ?: d)` in /var/www/virtual/invninja/html/app/Models/Account.php on line 1126"
```
What do you think about just sticking to 7.3 for now?

Reply:
Even with 7.3 it has some problems.
I have it running here with 7.3 and my interface is broken, and I can't view the already generated invoices.
@DaniW42 4.5.31 isn't supported and don't receive any update in the future, so it would probably just better to focus on v5 because they are already in r2 and people use it in Production.
I tried to install v5 but the composer throwing out an error:
Script vendor/bin/snappdf download handling the post-install-cmd event returned with error code 126